releasenotes/notes/0.7/prepare-0.7.3-86eeb02864cf60b0.yaml

```diff
@@ -0,0 +1,12 @@
+---
+prelude: |
+  The 0.7.3 release fixes the circuit cutting code to be compatible
+  with the forthcoming Qiskit 1.2 release.  It also pins numpy to
+  version 1.X, as DOCplex (which is used by the legacy CutQC
+  implementation) is incompatible with numpy 2.0.
+fixes:
+  - |
+    The circuit cutting module now uses a stringified UUID instead of
```

Comment:
Is this a breaking change? Is this something that surfaces to the user level?

Reply:
OK sorry, this is a "fix", not an "upgrade". Disregard

Reply:
> Is this something that surfaces to the user level?

It does not surface to the user level.  The uuid stuff is all internal to the `separate_circuit` function.